### ISSUE TITLE
Handling nonce in extrinsics so it's possible to send multiple extrinsics in parallel

### ISSUE THREAD

**Issue by TvrtkoM** [zeitgeistpm/ui#690]
![image](https://user-images.githubusercontent.com/551965/218440385-0fb7f08e-ea17-4804-839e-7f321885ce7b.png)

When trying to quickly redeem the outcomes from portfolio page without waiting the previous transaction to complete, redeem buttons become unresponsive with the following error:

```
CORE: submitAndWatchExtrinsic(extrinsic: Extrinsic): ExtrinsicStatus:: 1014: Priority is too low: (625 vs 625): The transaction has too low priority to replace another transaction already in the pool.
```

suggested fix is to manage the nonce in the app somehow

**Comment by Robiquet** [comment]
Should be straight forward now that we use `useExtrinsic` everywhere
